linuxNew/jdk/debian/src/main/packaging/temurin/11/debian/changelog

```diff
@@ -1,4 +1,8 @@
-temurin-11-jdk (11.0.13.0.0+8-1) UNRELEASED; urgency=medium
+temurin-11-jdk (11.0.14.0.0+9-1) UNRELEASED; urgency=medium
```

Comment:
This is part of an individual entry so there should be two, rather than a replace. See example: https://github.com/romlok/python-debian/blob/master/debian/changelog

Reply:
Updated accordingly.